_posts/2015-07-08-employee-induction-programme.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: "How to welcome new employees"
+date: "2015-07-08 06:02:32"
+icon: sign-in
+tags: induction employee welcome introduction onboarding
+categories: jobs
+authors: ["Birgit Bader"]
+---
+
+Most likely, everyone already has made their experience with being oriented to a new organisation.
```

Comment:
instead of "everyone" I would prefer something like "everyone reading this". And is "oriented" correct here?